packages/components/src/FormatFile/FormatFile.css

```diff
@@ -1,10 +1,6 @@
-:root {
-  --file-card--height: calc(var(--space-base) * 3.5);
-  --file-card--background-color: var(--color-white);
-  --file-card--header-background-color: var(--color-greyBlue--lighter);
-  --file-card--content-background-color: var(--color-white);
+.formatFile {
+  --file-card--background-color: var(--color-surface);
```

Comment:
A few unused variables in here removed (and one that was used only once which doesn't actually merit using a var?)

Reply:
No usages in Jobber

Reply:
I see below we replaced `var(--file-card--border-color)` with `var(--color-border)`

that appears to have been the only instance of `--file-card--border-color`

if that's how we'd prefer to define the border color, then we can get rid of the variable definition altogether since it's not being used!